packages/components/src/components/div100vh-container/div100vh-container.tsx

```diff
@@ -17,8 +17,8 @@ import Div100vh from 'react-div-100vh';
 type TDiv100vhContainerProps = {
     id?: string;
     height_offset?: string;
-    is_bypassed: boolean;
-    is_disabled: boolean;
+    is_bypassed?: boolean;
+    is_disabled?: boolean;
```

Comment:
@likhith-deriv Should we give them a default value of `false` at lines `30` and `31`? 🤔

Reply:
To be more explicit we can. Else since they are boolean, undefined becomes false

Reply:
`undefined` becomes `false` in JS world, In TS world it’s `true | false | undefined` 😁